config/betamocks/services_config.yml

```diff
@@ -73,6 +73,13 @@
       :uid_location: header
       :uid_locator: 'va_eauth_pnid'
 
+  # Claims
+  - :method: :get
+    :path: <%= "/wss-claims-services-web-#{Settings.evss.versions.claims}/rest/vbaClaimStatusService/getClaims" %>
+    :file_path: "evss/claims/index"
+    :cache_multiple_responses:
+      :uid_location: header
```

Comment:
Does `header` need to be a string here or does yaml coerce it into one anyway so it doesn't matter?